docs/man_pages/project/configuration/generate-splash.md

```diff
@@ -0,0 +1,34 @@
+generate splashscreen
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder generate splashscreen <Path>` [--force]
+
+Generates splashscreens from a single high-resolution image and saves them to the current project
```

Comment:
Generates splash screens for all mobile platforms from a single high-resolution image and saves them to the `App_Resources` folder of the current project.